mediachain/transactor/block_cache.py

```diff
@@ -24,8 +24,9 @@ def open_db(path):
         opts = rocksdb.Options(create_if_missing=True)
         return rocksdb.DB(path, opts)
     except ImportError:
-        print('using in-memory blockchain cache. please install '
-              'rocksdb to enable persistent cache')
+        logger = get_logger(__name__)
+        logger.info('using in-memory blockchain cache. please install '
```

Comment:
maybe this should be a warning so that the user actually sees it with default config?